site/docs/Learn-Schema.md

```diff
@@ -85,7 +90,7 @@ That means that the GraphQL service needs to have a `Query` type with `hero` and
 ```graphql
 type Query {
   hero(episode: Episode): Character
-  droid(id: ID!): Droid
+  droid(id: String!): Droid
```

Comment:
Yeah I noticed that the old example schema had String instead of ID, but let's upgrade.